cmd/junit2jira/main.go

```diff
@@ -151,6 +152,28 @@ func run(p params) error {
 	return errors.Wrap(j.createHtml(jiraIssues), "could not create HTML report")
 }
 
+func (j junit2jira) getMergedFailedTests(testSuites []junit.Suite) ([]j2jTestCase, error) {
+	failedTests, err := testcase.GetFailedTests(testSuites)
+	if err != nil {
+		return nil, errors.Wrap(err, "could not get failed tests")
+	}
+	log.Infof("Found %d failed tests", len(failedTests))
+
+	failedJ2jTests := make([]j2jTestCase, 0, len(failedTests))
+	for _, failedTest := range failedTests {
+		failedJ2jTests = append(failedJ2jTests, newJ2jTestCase(failedTest, j.params))
+	}
+
+	if len(failedTests) > j.threshold && j.threshold > 0 {
```

Comment:
I think this reads more naturally:
```suggestion
	if j.threshold > 0 && len(failedTests) > j.threshold {
```

Reply:
I find the first option easier to understand. I changed it anyway to:
```suggestion
	if 0 < j.threshold && j.threshold < len(failedTests) {
```
- `j.threshold` is between `0` and `len(failedTests)`